src/entities/add-proof-entity.ts

```diff
@@ -0,0 +1,26 @@
+import { Entity, Column, PrimaryColumn, ManyToOne, JoinColumn } from "typeorm";
+import { HeaderEntity } from "./header-entity";
+
+/*
+  Schema
+    header_id   VARCHAR(64) PRIMARY KEY REFERENCES node_headers (id),
+    proof_bytes VARCHAR NOT NULL,
+    digest      VARCHAR NOT NULL
+    PRIMARY KEY (header_id)
+*/
+
+@Entity({ name: "node_add_proofs" })
```

Comment:
This entity is called node_**ad**_proofs, please change this file to reflect this.